### ISSUE TITLE
API 400: You provided an invalid order type: OrderType.FULL_OPEN, please set it to be FULL_RESTRICTED or PARTIAL_RESTRICTED

### ISSUE THREAD

**Issue by theboxer** [ProjectOpenSea/opensea-js#577]
When trying to create a buy or sell order I'm getting this error response: `{"errors":["You provided an invalid order type: OrderType.FULL_OPEN, please set it to be FULL_RESTRICTED or PARTIAL_RESTRICTED"]}`

```js
seaport.createSellOrder({
  asset: {
    tokenId,
    tokenAddress,
    schemaName,
  },
  accountAddress,
  startAmount,
  expirationTime,
});

```

**Comment by plondon** [comment]
You need to set `restrictedByZone: true` and `allowPartialFills: false`

**Comment by theboxer** [comment]
@plondon These are not settable through `createSellOrder`: https://github.com/ProjectOpenSea/opensea-js/blob/master/src/sdk.ts#L1030

**Comment by plondon** [comment]
https://github.com/ProjectOpenSea/opensea-js/pull/598

**Comment by jtaaa** [comment]
Closing this out. Fixed in version 4.0.3
